spec/order_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../app/lib/order'
+
+describe 'an order' do
+
+  before(:each) do
+    @order = Order.new
```

Comment:
Use `let` or even better `subject(:order)` when describing the object which is the subject of your tests